README.md

```diff
@@ -84,40 +85,44 @@ weave launch-router \
   $KUBE_WORKER_1 $KUBE_WORKER_2
 ```
 
-If you only know some of the peer hostnames/IPs, you can launch Weave Net router like this:
+If only some of the peer hostnames/IPs are known, launch the Weave Net router as follows:
 
 ```Shell
 weave launch-router --init-peer-count 6 $KUBE_ETCD_1 $KUBE_MASTER_0 $KUBE_WORKER_1
 ```
 
-or even
+or:
 
 ```Shell
 weave launch-router --init-peer-count 6 $KUBE_MASTER_0
 ```
 
-Otherwise, you could also run `weave connect` later, just make sure to pass `--init-peer-count` with a number of servers you are expecing to have in your cluster.
+You can also run `weave connect` at a later time, and pass `--init-peer-count` with the number of servers you expect to have in your cluster.
 
-Next, you need to launch Weave proxy for the Docker API. It's crictical to pass `--rewrite-inspect` flag for Kubernetes integration to function properly.
+In order to communicate with the docker daemon through Weave, launch Weave Docker API Proxy. 
 
 ```Shell
 weave launch-proxy --rewrite-inspect
 ```
 
-And finally, you need to expose the host on Weave Net and set a DNS record for it like so:
+>>**Important** For the Kubernetes integration to function properly, it is crictical that you pass the `--rewrite-inspect` flag when launching the Weave Docker API Proxy.
+
+
+Finally, expose the host and make it available to Weave Net by setting its DNS record:
 
 ```Shell
 weave expose -h "$(hostname).weave.local"
 ```
 
-Before launching any containers you will also need to point Docker client to Weave proxy socket by setting `$DOCKER_HOST` like with the following command:
+Before launching any containers you must first point the Docker client to the Weave proxy socket by setting `$DOCKER_HOST` environment variable:
+
 ```Shell
 eval $(weave env)
 ```
 
-The above can be wrapped in a simple provisioning script which you will find below.
+The above can be wrapped in a convenient provisioning script which you can find described below.
```

Comment:
let's turn this into "you can find what such script may look like some of examples" and link to "Further Examples" section below.